packages/components/src/index.js

```diff
@@ -16,6 +16,7 @@ export { default as ClipboardButton } from './clipboard-button';
 export { default as ColorIndicator } from './color-indicator';
 export { default as ColorPalette } from './color-palette';
 export { default as ColorPicker } from './color-picker';
+export { default as CustomSelect } from './custom-select';
```

Comment:
Why it's named CustomSelect and not just Select

Reply:
We already have `SelectControl` so that felt confusing, but I don't have a strong opinion on it.

Reply:
Good remark, I do think in that case that it should be SomethingControl. `CustomSelectControl` maybe. I don't like Custom that much though but I also don't have a lot of ideas.

Reply:
I prefer `Select` to be honest. The goal is for it to be THE React version of `select` so that makes sense.

`Control` makes it seem like it uses `BaseControl` which it doesn't.

Reply:
I think it would be good to keep the "Custom", or some similar indication that this is not a standard HTML select (which we should also continue to provide, for cases where the options don't need to be visually distinguishable from one another)